src/runtime_src/xdp/profile/core/rt_profile.h

```diff
@@ -147,7 +147,7 @@ namespace xdp {
 
     // Log device trace
     void logDeviceTrace(std::string deviceName, std::string binaryName, xclPerfMonType type,
```

Comment:
I know this isn't changed in this PR, but please pass strings as const std::string& when then are constants that remain unchanged in function

Reply:
Thanks for pointing this out! I found a cleaned this up in a lot of places in our code.